.github/workflows/ci_action.yml

```diff
@@ -0,0 +1,26 @@
+name: CI test
+on:
+  pull_request:
+    types: [labeled]
+jobs:
+
+  ci-test:
+    if: ${{ github.event.label.name == 'ci-test' }}
+    runs-on: ubuntu-latest 
+    steps:
+      - name: Checkout 
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: install dependencies and run tests
+        run: |
+          sudo add-apt-repository ppa:deadsnakes/ppa
+          sudo apt-get update
+          sudo apt install sox
+          sudo apt-get install -y python3.7
+          sudo apt-get install -y python3.7-venv 
+          python3.7 -m venv venv
+          source venv/bin/activate
+          pip install torch torchvision torchaudio
+          pip install -e .
+          pytest
```

Comment:
do we test the package from src of do we test it from tests folder? what is the best practice?

Reply:
We test it from the root of the repo.
I think its the best practice since pytest searches recursively for all .py files with test* prefix and runs them.